kubecost/templates/_helpers.tpl

```diff
@@ -548,21 +548,24 @@ federated storage config helpers
   {{- end }}
 {{- end -}}
 
-{{- define "kubecost.federatedStorage.config" }}
+{{/*
+NOTE: added kubecostModel for backward compatibility
+*/}}
+{{- define "kubecost.federatedStorage.config-b64" }}
```

Comment:
I'm not married to this change with including the type in the name. But it does make it obvious that it should be opaque.